CHANGELOG.md

```diff
@@ -1,3 +1,78 @@
+# 1.17.0-rc1 / 2025-04-28
+
+* [FEATURE] Add Kotlin Compiler Plugin environment.
+  See [#314](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/314)
+* [FEATURE] Add `ComposeTagTransformer` to inject `datadog` modifier call.
+  See [#321](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/321)
+* [FEATURE] Register plugin to Kotlin compile options.
+  See [#322](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/322)
+* [FEATURE] Implement annotation mode for instrumentation.
+  See [#342](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/342)
+* [FEATURE] Implement `NavHost` instrumentation transformer.
+  See [#329](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/329)
+* [FEATURE] Implement Kotlin Compiler Plugin DSL configuration.
+  See [#330](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/330)
+* [FEATURE] Add `semantics role` for `Image`, `Icon`, and `AsyncImage`.
+  See [#334](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/334)
+* [FEATURE] Use composable function name as the semantics tag.
+  See [#337](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/337)
+* [IMPROVEMENT] Filter out `ksp` and `KaptGenerateStubsTask` from `
+  freeCompilerArgs`. See [#335](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/335)
```

Comment:
not sure if formatting will be right here by having a new line after opening backtick

```suggestion
* [IMPROVEMENT] Filter out `ksp` and `KaptGenerateStubsTask` from `freeCompilerArgs`. See [#335](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/335)
```